simplivity/resources/resource.py

```diff
@@ -223,7 +223,7 @@ def do_get(self, uri):
         """
         return self._connection.get(uri)
 
-    def do_post(self, uri, data, timeout, custom_headers, flags=None):
+    def do_post(self, uri, data, timeout, custom_headers=None, flags=None):
```

Comment:
args should be listed in the order that they are defined since they could be positional

doc string needs to be updated.

            uri: URI of the resource.
            data: Request body of the call
            timeout: Time out for the request in seconds.
            custom_headers: Allows to add custom http headers.
            flags: Dictionary of filters, example: {'name': 'name'}

Reply:
Updated the docstring as suggested